src/api/README.md

```diff
@@ -2187,6 +2192,118 @@ Result:
 ```
 
 
+## Notes APIs
+
+### Get all Notes
+
+API sets: `READ`
+
+```
+URI: /api/v2/notes
+Method: GET
+```
+
+Example:
+
+```sh
+curl -H 'Content-Type: application/json' http://127.0.0.1:6420/api/v2/notes
+```
+
+Result:
+
+```json
+[
+    {
+        "txid": "62b1e205aa2895b7094f708d853a64709e14d467ef3e3eee54ef79bcefdbd4c8",
+        "notes": "Paycheck for Mr. Mueller"
+    },
+    {
+        "txid": "9c8995afd843372636ae66991797c824e2fd8dfffa77c901c7f9e8d4f5e87114",
+        "notes": "A Note \n"
```

Comment:
Maybe the API should trim whitespace from notes

Reply:
I don't see any advantage in doing so. I think everyone should be fully capable on what is written in the notes.